src/app/components/single-select/single-select.js

```diff
@@ -8,7 +8,8 @@ import {template} from './single-select.hbs';
 @props({
     template,
     regions: {
-        optionList: '.option-list'
+        optionList: '.option-list',
+        hasDropdown: '.has-dropdown'
```

Comment:
Are you doing anything with this region? A region is typically for putting sub-views into.